charts/console/values.go

```diff
@@ -220,6 +193,6 @@ type Creatable struct {
 type Image struct {
 	Registry   string            `json:"registry"`
 	Repository string            `json:"repository"`
-	PullPolicy corev1.PullPolicy `json:"pullPolicy"`
+	PullPolicy corev1.PullPolicy `json:"pullPolicy" jsonschema:"required,pattern=^(Always|Never|IfNotPresent)$,description=The Kubernetes Pod image pull policy."`
```

Comment:
There's no need to include the pattern or description here, that'll be inferred from the `corev1.PullPolicy` type when the schema is generated.

Reply:
Hmm I included this as the redpanda chart has the same thing for image pull policy ([code](https://github.com/redpanda-data/redpanda-operator/blob/main/charts/redpanda/values.go#L132))?

Reply:
Ah, it shouldn't be there either 😓